tests/helpers.test.js

```diff
@@ -174,20 +173,40 @@ describe('Helper Tests', () => {
     })
 
     describe('addScript', () => {
-        it('should add script to package.json if package.json exits', () => {
-            fs.readFileSync.mockReturnValue(`{"scripts": {} }`)
-            console.error = jest.fn()
-            console.log = jest.fn()
-            addScript('test', 'node')
-            expect(console.error).not.toBeCalled()
-            expect(console.log.mock.calls[0][0]).toContain('test script into package.json')
-        })
-        it('throws an error if no package.json', () => {
-            fs.readFileSync.mockReturnValue(null)
+      it('should add script to package.json if package.json exits', () => {
```

Comment:
We should add a couple of tests to see what write file was called with to test the store.name/package.json vs ./package.json